packages/Operators/test/tstNodeToNodeOperator.cpp

```diff
@@ -173,12 +175,30 @@ void setupAndRunTest( const std::string &input_file,
 //---------------------------------------------------------------------------//
 // Tests.
 //---------------------------------------------------------------------------//
-TEUCHOS_UNIT_TEST( NodeToNodeOperator, node_to_node_test )
+TEUCHOS_UNIT_TEST( NodeToNodeOperator, matching_node_test )
+{
+    // Run the test.
+    Teuchos::Array<double> gold_data;
+    Teuchos::Array<double> test_result;
+    setupAndRunTest( "matching_node_test.xml", gold_data, test_result, false );
+
+    // Check the results.
+    TEST_EQUALITY( gold_data.size(), test_result.size() );
+    int num_points = gold_data.size();
+    for ( int i = 0; i < num_points; ++i )
+    {
+        TEST_FLOATING_EQUALITY( gold_data[i], test_result[i], epsilon );
+    }
+}
+
+//---------------------------------------------------------------------------//
+TEUCHOS_UNIT_TEST( NodeToNodeOperator, non_matching_node_test )
 {
     // Run the test.
     Teuchos::Array<double> gold_data;
     Teuchos::Array<double> test_result;
-    setupAndRunTest( "node_to_node_test.xml", gold_data, test_result );
+    setupAndRunTest( "non_matching_node_test.xml", gold_data, test_result,
+                     true );
```

Comment:
So I check and it does exists.  It is called [TEST_THROW](https://trilinos.org/docs/dev/packages/teuchos/doc/html/group__Teuchos__UnitTestAssertMacros__grp.html#ga8a6d1a22211e72a7eabe164cf3aa2103).  Please use it here and make sure the correct exception is thrown.